packages/jest-fake-timers/src/__tests__/jestFakeTimers.test.ts

```diff
@@ -40,6 +41,20 @@ describe('FakeTimers', () => {
       expect(global.setTimeout).not.toBe(undefined);
     });
 
+    it('accepts to promisify setTimeout mock', () => {
+      const global = ({process} as unknown) as NodeJS.Global;
+      const timers = new FakeTimers({
+        config,
+        global,
+        moduleMocker,
+        timerConfig,
+      });
+      timers.useFakeTimers();
+      const promisifiedTimeout = util.promisify(global.setTimeout)(0);
+      timers.runAllTimers();
+      expect(promisifiedTimeout).not.toBe(undefined);
```

Comment:
Sorry, GH lost a comment here last time -

This does not test that the Promise actually resolves, we'll need to check that.

Reply:
Do you have any idea about how we can test this? The custom timeout does not call any callback passed into it.

I've thought about checking the promise result with the extra parameters.

Reply:
I imagine something like: set the timeout with an arg of `42` and after `runAllTimers` do `await expect(timeoutPromise).resolves.toBe(42)`

Reply:
Thanks for the feedback. It's ok.